docs/connecting-data/connecting-to-apis-and-external-data/donorsearch-api.md

```diff
@@ -2,6 +2,91 @@
 description: How to Connect DataDistillr to the DonorSearch API
 ---
 
-# DonorSearch API
+# Connecting to DonorSearch
 
-Details coming soon!
\ No newline at end of file
+## How to Connect DataDistillr to DonorSearch
+To set up a data source connection for DonorSearch, you will need to have:
+
+- A [unique name](#name) for your data source connection to be used in queries
+- The [API Key](#api-key) generated by your DonorSearch account
```

Comment:
Missing API Key section

Reply:
Understand we may not have the info, but provide a link to the site if we are saying they need this information so it doesn't just look like we're dropping them off with no help